card/db/migrate_core_cards/20150501010515_responsive_sidebar.rb

```diff
@@ -0,0 +1,11 @@
+# -*- encoding : utf-8 -*-
+
+class ResponsiveSidebar < Card::CoreMigration
+  def up
+    if (layout = Card.fetch 'Default Layout') &&
```

Comment:
Use a guard clause instead of wrapping the code inside a conditional expression.